clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -1952,10 +1988,6 @@ private void maybeThrowFencedInstanceException() {
         }
     }
 
-    private void maybeInvokeCommitCallbacks() {
-        offsetCommitCallbackInvoker.executeCallbacks();
-    }
-
```

Comment:
Any reason we don't want to keep this method abstraction?

Reply:
I do not insist 😄 I was just curious and I agree that it doesn't odd anything in this case.